tests/test-xdp-utils.c

```diff
@@ -170,6 +165,14 @@ test_app_id_via_systemd_unit (void)
   app_id = _xdp_parse_app_id_from_unit_name ("app-com.obsproject.Studio-d70acc38b5154a3a8b4a60accc4b15f4.scope");
   g_assert_cmpstr (app_id, ==, "com.obsproject.Studio");
   g_clear_pointer (&app_id, g_free);
+
+  app_id = _xdp_parse_app_id_from_unit_name ("app-foo-bar.scope");
+  g_assert_cmpstr (app_id, ==, "foo");
```

Comment:
Might be more obviously correct if you used `app-firefox-jcfppqx.scope`, which is functionally equivalent.